[TH]_Psycho_Pass_THE_MOVIE.ass

```diff
@@ -169,66 +181,66 @@ Dialogue: 0,0:16:07.67,0:16:09.01,Default,,0,0,0,,I see...
+Dialogue: 0,0:18:43.58,0:18:46.96,Default,,0,0,0,,ผู้สังเกตการณ์ชิโมสึกิกำลังปฏิบัติหน้าที่ภายใต้คำสั่งของประธาน
+Dialogue: 0,0:18:47.46,0:18:48.75,Default,,0,0,0,,เธอทำอะไร ?
+Dialogue: 0,0:18:49.04,0:18:51.67,Default,,0,0,0,,ชั้นขัดคำสั่งผู้บังคับบัญชาไม่ได้
+Dialogue: 0,0:18:51.75,0:18:53.71,Default,,0,0,0,,เอาเป็นว่ามาที่นี่ให้เร็วที่สุดนะ
+Dialogue: 0,0:18:53.84,0:18:55.71,Default,,0,0,0,,ถึงจะไม่รู้ว่ากำลังจะทำอะไรก็เถอะ
+Dialogue: 0,0:18:55.84,0:18:57.22,Default,,0,0,0,,หยุดเธอให้หน่อยได้มั้ย ?
+Dialogue: 0,0:18:58.84,0:19:00.30,Default,,0,0,0,,ถ่วงเวลาให้ก็ยังดี
+Dialogue: 0,0:19:00.72,0:19:02.93,Default,,0,0,0,,ไม่ไหว ไม่ทันเวลาแน่ๆ
+Dialogue: 0,0:19:03.68,0:19:05.18,Default,,0,0,0,,ขอโทษจริงๆ
+Dialogue: 0,0:19:08.98,0:19:11.02,Default,,0,0,0,,เธอใช้วิธีดูความทรงจำ (Memory Scoop) งั้นเหรอ ?
+Dialogue: 0,0:19:11.36,0:19:12.77,Default,,0,0,0,,ตั้งใจจะทำอะไรกันแน่
+Dialogue: 0,0:19:13.07,0:19:15.03,Default,,0,0,0,,อย่ามาลงที่ชั้นสิ
+Dialogue: 0,0:19:15.53,0:19:17.32,Default,,0,0,0,,คำสั่งท่านประธานค่ะ
+Dialogue: 0,0:19:17.74,0:19:20.61,Default,,0,0,0,,มันไม่ใช่ความผิดของนักวิเคราะห์คาราโนโมริหรอกค่ะ
+Dialogue: 0,0:19:23.16,0:19:25.95,Default,,0,0,0,,ผู้ต้องหาเป็นอาชญากรแฝงนะคะ
+Dialogue: 0,0:19:26.12,0:19:27.62,Default,,0,0,0,,เพียงแค่ไม่มีเคสแบบนี้ก่อนหน้า
+Dialogue: 0,0:19:27.75,0:19:29.33,Default,,0,0,0,,ไม่ได้แปลว่าผิดกฏหมายนะคะ
+Dialogue: 0,0:19:41.22,0:19:42.39,Default,,0,0,0,,โหดร้าย
+Dialogue: 0,0:19:43.39,0:19:45.60,Default,,0,0,0,,เพื่อที่จะให้คลื่นสมองออกมาเด่นชัด
+Dialogue: 0,0:19:45.72,0:19:47.85,Default,,0,0,0,,เราจึงให้ยาเค้าค่ะ
+Dialogue: 0,0:19:48.60,0:19:52.48,Default,,0,0,0,,เราไม่เคยมีผู้ลักลอบเข้าประเทศติดอาวุธมาก่อน
+Dialogue: 0,0:19:53.02,0:19:54.90,Default,,0,0,0,,และนอกจากนี้ยังเป็นอาชญากรแฝง
+Dialogue: 0,0:19:55.40,0:19:57.82,Default,,0,0,0,,เค้าไม่มีสิทธิอะไรที่เราจะต้องไปปราณีปรานอม
+Dialogue: 0,0:19:58.07,0:20:00.74,Default,,0,0,0,,ทั้งหมดนี้ทำให้ชั้นได้ข้อสรุปแล้วค่ะ
+Dialogue: 0,0:20:00.99,0:20:02.07,Default,,0,0,0,,ข้อสรุปเหรอ ?
+Dialogue: 0,0:20:02.20,0:20:04.74,Default,,0,0,0,,ท่านประธานเรียกค่ะ รุ่นพี่
+Dialogue: 0,0:20:06.16,0:20:08.79,Default,,0,0,0,,คุณให้ผู้สังเกตการณ์ชิโมสึกิทำอะไรลงไป
+Dialogue: 0,0:20:09.29,0:20:12.25,Default,,0,0,0,,สอบสวนสิ่งที่เกิดยังไงล่ะ
+Dialogue: 0,0:20:11.00,0:20:12.25,Default,,0,0,0,,{\an8}{\3c&H20116C&}ไม่รู้สึกละอายบ้างเหรอ
+Dialogue: 0,0:20:12.33,0:20:14.17,Default,,0,0,0,,มันก็เกิดขึ้นเหมือนทุกที
+Dialogue: 0,0:20:14.38,0:20:16.59,Default,,0,0,0,,เราแค่ทำในเรื่องที่จำเป็นเท่านั้นแหละ
+Dialogue: 0,0:20:17.13,0:20:18.67,Default,,0,0,0,,ดูนี่ซะก่อน
+Dialogue: 0,0:20:25.43,0:20:29.10,Default,,0,0,0,,ถ้าเป็นเธอล่ะก็ คงรู้จักเค้าคนนี้สินะ
+Dialogue: 0,0:20:30.18,0:20:32.89,Default,,0,0,0,,เค้าเคยเป็นผู้ใต้บังคับบัญชาของเธอ
+Dialogue: 0,0:20:38.69,0:20:42.45,Default,,0,0,0,,กลุ่มผู้ก่อการร้ายข้ามชาติลักลอบเข้ามา
+Dialogue: 0,0:20:42.61,0:20:47.62,Default,,0,0,0,,และในความทรงจำของเค้ามีเจ้าหน้าที่ปฎิบัติการที่หลบหนีไปเมื่อสามปีก่อน
```

Comment:
ก่อนหน้านี้บอกว่า กลุ่มก่อการร้ายก็จริง
แต่เหตการณ์จริงคือ มันจับคน คนเดียว มาดูความทางจำฮะ แล้วญี่ปุ่นมันใช้คำว่า sono ชี้นามแต่หนึ่งเดียว โดยเฉพาะเจาะจง เหมือน That ในภาษาอังกฤษฮะ